app/helpers/application_helper.rb

```diff
@@ -69,6 +69,7 @@ def contributors
       { name: "Andrew Harvey", email_md5: "7284ad488e18a2b052a9c7b8fe1e3922", github: "andrewharvey" },
       { name: "Andrew Perry", email_md5: "7d329af0dcfe18c8797f542938286e46", github: "andrewperry" },
       { name: "Ben M", email_md5: "a99b7918f974feb902e89bb613b32991", github: "" },
+      { name: "Billy Kwong", email_md5: "a99b7918f974feb902e89bb613b32991", github: "tuppa" },
```

Comment:
Oops @tuppa I just noticed that you used the same email_md5 as "Ben M" which means that the gravatar that shows up is the same as his. Any chance that you could use the md5 of your email address or if you don't want to do that the md5 hash of a made-up email address? Thanks!

Reply:
Ooops sorry about that, here's 2nd try:
https://github.com/openaustralia/planningalerts/pull/1442